docs/changelog.rst

```diff
@@ -27,6 +27,10 @@ Changed
 
   This is a more standard file name for environment variables than the previous ``env.local``. For now we'll still load from the old file too, but a warning will be displayed to rename the file.
 
+* **Breaking change**
```

Comment:
The changelog is a list inside each header, but I'll fix this in a separate commit, noticed one other thing to add too 👍